modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: REFERENCE
+
+[id="ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds_{context}"]
+= Integration between shared resources, Insights Operator, and OpenShift Builds
+
+[role="_abstract"]
+Integration between shared resources, Insights Operator, and OpenShift Builds makes using Red Hat subscriptions (RHEL entitlements) easier.
```

Comment:
```suggestion
Integration between Shared Resources, Insights Operator, and OpenShift Builds makes using Red Hat subscriptions (RHEL entitlements) easier in {product-title} Builds.
```

Reply:
The term, "shared resources," uses sentence case throughout the OpenShift docs. I've updated "OpenShift Builds" to "{product-title} Builds." Thanks.